core/cmd/shell_local.go

```diff
@@ -822,6 +825,49 @@ func dropDanglingTestDBs(lggr logger.Logger, db *sqlx.DB) (err error) {
 	return
 }
 
+type failedToRandomiseTestDBSequencesError struct{}
+
+func (m *failedToRandomiseTestDBSequencesError) Error() string {
+	return "failed to randomise test db sequences"
+}
+
+// randomiseTestDBSequences randomizes sequenced table columns sequence
+// This is necessary as to avoid false positives in some test cases.
+func randomiseTestDBSequences(db *sqlx.DB) error {
```

Comment:
```suggestion
// randomizeTestDBSequences randomizes sequenced table columns sequence
// This is necessary as to avoid false positives in some test cases.
func randomizeTestDBSequences(db *sqlx.DB) error {
```

I am English so the "s" spelling seems natural to me also, but the language of code and this repo is American English and best to stay consistent.